torch/fx/experimental/proxy_tensor.py

```diff
@@ -373,7 +387,8 @@ def try_set_proxy_slot(
     ) -> None:
         assert callable(proxy_callable)
         if isinstance(outer_s, SymInt):
-            set_proxy_slot(outer_s, tracer, thunkify(proxy_callable, outer_s, *args, **kwargs))
+            with enable_thunkify(tracer):
+                set_proxy_slot(outer_s, tracer, thunkify(tracer, proxy_callable, outer_s, *args, **kwargs))
```

Comment:
This is prophylactically setting up proxies for size/stride/etc fields on the Tensor, if they're never accessed we don't need to put them in